apim/3.x/templates/api/api-bridge-ingress.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.api.enabled -}}
+{{- if .Values.api.services.bridge.enabled -}}
+{{- if .Values.api.services.bridge.ingress.enabled -}}
+{{- $serviceAPIName := include "gravitee.api.fullname" . -}}
+{{- $serviceAPIPort := .Values.api.services.bridge.service.externalPort -}}
+{{- $ingressPath   := .Values.api.services.bridge.ingress.path -}}
+{{- $apiVersion := include "common.capabilities.ingress.apiVersion" . -}}
+apiVersion: {{ $apiVersion }}
+kind: Ingress
+metadata:
+  name: {{ template "gravitee.api.fullname" . }}-bridge
+  labels:
+    app.kubernetes.io/name: {{ template "gravitee.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/version: {{ .Values.api.image.tag | default .Chart.AppVersion | quote }}
+    app.kubernetes.io/component: "{{ .Values.api.name }}"
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+  annotations:
```

Comment:
@kamiiiel can you please check any of the other ingress, you'll see that there is something specific to do here.